testing/benchmark_hooks.py

```
@@ -0,0 +1,95 @@
+    end = time.perf_counter()
+    return end - start
+
+
+def benchmark():
+    results = {}
+    for hook in HOOKS:
+        times = []
+        print(f"Benchmarking {hook['name']}...")
+        for i in range(REPEATS):
+            # Clean up any changes before each run
+            subprocess.run(["git", "restore"] + TARGET_FILES)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)